pages/docs/configuration/configuration-acceleration.md

```diff
@@ -17,7 +17,8 @@ All data communicated within a coupling scheme needs to be configured through `e
 <coupling-scheme:serial-implicit>
```

Comment:
From this example, I understand that the `<data>` inside the `<acceleration>` does not mean "which data is accelerated", but "which data contributes to the coefficients".

This is not obvious, and we should clarify it somewhere.

Reply:
The config reference could be a bit more verbose though.

"compute the acceleration" can mean many things.